src/ducks/filters/scrollAware.js

```diff
@@ -37,7 +37,9 @@ export default Wrapped =>
       }
     }, 16)
 
-    render(props, { scrolling }) {
+    render() {
+      const props = this.props
+      const { scrolling } = this.state
```

Comment:
Should we do the same thing everywhere ? I think I saw some components using props and state from `render` arguments « the Preact way » while working on categorization pages

Reply:
Yes I think it would be good to do it step by step. For example enzyme only works with react.